source/elements/oneTBB/source/containers/concurrent_hash_map_cls/modifiers.rst

```diff
@@ -18,18 +18,44 @@ Inserting values
 
         bool insert( accessor& result, const key_type& key );
 
-    If the accessor ``result`` is not empty, releases the ``result`` and
+    If the ``result`` accessor is not empty, releases the ``result`` and
     attempts to insert the value constructed from ``key, mapped_type()`` into the container.
 
     Sets the ``result`` to provide access to the inserted element or to the element with equal key
-    that was already presented in the container.
+    which was already presented in the container.
 
     **Requirements**:
 
-    * the type ``value_type`` must meet the ``EmplaceConstructible`` requirements the from [container.requirements] ISO C++ Standard section.
-    * the type ``mapped_type`` must meet the ``DefaultConstructible`` requirements from the [defaultconstructible] ISO C++ Standard section.
+    * the ``value_type`` type must meet the ``EmplaceConstructible`` requirements the from [container.requirements] ISO C++ Standard section.
+    * the ``mapped_type`` type must meet the ``DefaultConstructible`` requirements from the [defaultconstructible] ISO C++ Standard section.
 
-    **Returns**: ``true`` if an element was inserted; ``false``, otherwise.
+    **Returns**: ``true`` if an element is inserted; ``false`` otherwise.
+
+--------------------------
+
+    .. code:: cpp
+
+        template <typename K>
+        bool insert( const_accessor& result, const K& key );
+
+        template <typename K>
+        bool insert( accessor& result, const K& key );
+
+    If the ``result`` accessor is not empty, releases the ``result`` and
```

Comment:
Better to remove indent here and later on, otherwise, it'll be built as part of the code block

Reply:
It seems like we use the same indent in such blocks and it renders correct